src/examples/ryansoffice/scene.js

```diff
@@ -0,0 +1,61 @@
+// SURGICAL PLANNING sample code
+// important: test in metachromium which enables transparent WebXR rendering on desktop!
+// https://github.com/webaverse/metachromium-bin
+
+import { AmbientLight, LoadingManager, Scene, Color, Mesh, Plane, DoubleSide, SphereBufferGeometry, PlaneBufferGeometry, MeshBasicMaterial, MeshNormalMaterial, MeshPhongMaterial, Object3D, HemisphereLight, DirectionalLight, SpotLight, ShaderMaterial, AdditiveBlending, BufferGeometry, TextureLoader, Float32BufferAttribute, Points, DynamicDrawUsage, Vector3 } from "three";
+import { GLTFLoader } from "three/examples/jsm/loaders/GLTFLoader"
+import { camera } from "../../engine/engine"
+import { Renderer } from "../../engine/renderer"
+import { XRInput } from "../../engine/xrinput";
+import { State } from "../../engine/state";
+import { PeerConnection } from '../../engine/networking/PeerConnection'
+
+const GLTFOffice = require("./assets/models/office/ryans-home-office.gltf");
+const GLTFOfficeBin = require("./assets/models/office/ryans-home-office.bin");
+const GLTFOfficeJpg = require("./assets/models/office/ryans-home-office.jpg");
+
+export const scene = new Scene();
+//const networking = new PeerConnection(scene);
+
+scene.init = () =>
+{
+    var light = new AmbientLight( 0x404040 ); // soft white light
+    light.intensity = 10;
+    scene.add( light );
+
+    const manager = new LoadingManager();
+    const oldResolve = manager.resolveURL;
+    manager.resolveURL = (uri) => {
+        console.log("Resolving uri: " + uri);
+
+        // When the GLTF loader reads "ryan-home-office.jpg" in the GLTF json
+        // it thinks it is in the directory as the gltf file (i.e. /assets/models/)
+        // However, in webpack.common.js we put images in /assets/images
+        // so we need to point the image at the appropriate path
+        if (uri.includes("ryans-home-office.jpg"))
```

Comment:
Had to create a custom manager to resolve the url for the gltf image assets.

Reply:
this is going to be an ongoing problem with all gltf texture maps, let's discuss. Also going to research how it's done in other places.